### PR TITLE
More lenient preview resolution selection

### PR DESCRIPTION
**Pull request by a-szotyori**
Loosened the preview resolution to picture resolution aspect ratio matching. On devices like the HTC One E8 or HTC One M9 no matching preview resolution could be found.

### How to test
Run the tests in `ginivision/src/test`.

You can also run the `ginivision/src/androidTest/java/net/gini/android/vision/requirements/GiniVisionRequirementsTest.java`on a device to, but since all our device worked before this can be omitted.

### Merging
Merge after PR #28. `SizeSelectionHelper#getLargestSizeWithSameAspectRatio()`was renamed to `getLargestSizeWithSimilarAspectRatio()`and a test from #28 needs to be updated to use the new name.

### Ticket 
No ticket.

### CODE REVIEW

File: ginivision/src/test/java/net/gini/android/vision/internal/camera/api/SizeSelectionHelper_GetLargestSizeWithSameAspectRatioTest.java [hunk 1]
```diff
@@ -25,27 +25,43 @@
...
                 {"Largest 14:9 from decreasing resolutions", DECREASING_RESOLUTIONS,
                         new int[]{376, 240}, new int[]{3200, 2048}},
                 {"Largest 14:9 from increasing resolutions", INCREASING_RESOLUTIONS,
                         new int[]{376, 240}, new int[]{3200, 2048}},
                 {"Largest 14:9 from unsorted resolutions", UNSORTED_RESOLUTIONS,
-                        new int[]{376, 240}, new int[]{3200, 2048}}
+                        new int[]{376, 240}, new int[]{3200, 2048}},
+                // No exact matching aspect ratio for 4224x3136 (~1.346939) but should still find
+                // a similar resolution
+                {"Largest similar for 4224x3136 from decreasing resolutions", DECREASING_RESOLUTIONS,
```
Comment (nsobadjiev) [inline ginivision/src/test/java/net/gini/android/vision/internal/camera/api/SizeSelectionHelper_GetLargestSizeWithSameAspectRatioTest.java]: Nice! New test cases with the edge case.
